defaults/main.yml

```diff
@@ -24,24 +29,27 @@ postgres_labels:
   kubernetes.io/cluster-service: "true"
 
 postgres_volumes:
+    # PGDATA volume
     - name: varlibpostgres
-      hostPath: 
-        path: /var/lib/postgres/data
+      hostPath:
+        path: /var/lib/postgresql/data
```

Comment:
They two slave from same 'cluster' will run in different rc-name
Two slave from two different cluster should run in different domains name or it won't work. 

I prefer to add the domain-name in the path as it will isolate better and allow to run more than one cluster. It's not optimal but at least we can do a PG cluster per domain without collision